src/chain_sync/sync_state.rs

```diff
@@ -26,6 +27,7 @@ pub enum SyncStage {
     Messages,
     /// `ChainSync` completed and is following chain.
     Complete,
+    #[cfg_attr(test, arbitrary(skip))]
```

Comment:
This wasn't in the manual implementation. Not sure if it's a bug to omit. Not sure why you'd not use a `anyhow::Result<SyncStage>` for this in any case...